utils/generate-domains-blocklists/generate-domains-blocklist.py

```diff
@@ -304,6 +309,11 @@ def blocklists_from_config_file(
 
 args = argp.parse_args()
 
+whitelist = args.whitelist
+if whitelist:
+    print('Use of -w WHITELIST has been removed. Please use -a ALLOWLIST instead.')
```

Comment:
I don't think you need to put `WHITELIST` here, since in both cases the type of list the user is linking is the same. I think it would be fine to just say `The option to provide a set of names to exclude from the blocklist has been changed from -w to -a.` - this way it is consistent with the help text.

Also printing the help text before you exit (`argp.print_help()`) might be friendly too.